### PR TITLE
Ft user sign up 164027176

### PR DESCRIPTION
**Pull request by sengayire**
**What does this PR do?**
- create a new user account

***Description of Task to be completed?***
 - A user should create an account after being authenticated
**How should this be manually tested?**
- This can be tested by cloning locally this repo and use postman for the test or visit this link:  https://politico-andela.herokuapp.com/

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-multi-assign */
+import uuid from 'uuid';
+import sqlQueries from '../models/database/queries';
+// import connect from '../models/database/database';
```
Comment (FredDominant) [inline server/controllers/user.js]: delete this commented out part.

File: server/controllers/user.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      firstName,
+      lastName,
+      otherName,
+      email,
+      password,
+      phoneNumber,
+      passportUrl,
+    ];
+    const search = sqlQueries.selectAll;
+    let get = [];
```
Comment (FredDominant) [inline server/controllers/user.js]: I think this can be renamed better. `get` isn't a good name.

File: server/routes/user.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import user from '../controllers/user';
+
+const userRoute = express.Router();
+// userRoute.post('/userTable', user.createTable);
```
Comment (FredDominant) [inline server/routes/user.js]: remove this commented out code too

**Review by FredDominant — CHANGES_REQUESTED** [review]
You should address the feedback and then merge.

**Comment by sengayire** [comment]
marge after implement feedback

**Pull request merged**